src/sass/screen.scss

```diff
@@ -271,17 +273,17 @@ h3 {
 
 	}
 }
-.founding-team, .non-profits-wrap, .volunteers-wrap{
-	max-width: 55em;
-	margin: 0 auto;
-	padding: 0;
-	@media (min-width: 600px) {
-		padding:0;
-		.col-3 {
-			min-height: 15em;
-		}
-	}
-}
+// .founding-team, .non-profits-wrap, .volunteers-wrap{
```

Comment:
Are we better off removing this code if it's being commented out and not breaking anything or being used anywhere?